lib/tasks/single_run.rake

```diff
@@ -52,4 +52,17 @@ namespace :single_run do
       application.save
     end
   end
+
+  desc '2017-04-05: Set non-mentor favs to nil'
+  task set_non_mentor_favs_to_nil: :environment do
+    Application.rateable.each do |application|
+      fav1 = application.application_data['mentor_fav_project1']
+      fav2 = application.application_data['mentor_fav_project2']
+      if [fav1, fav2].include?('false')
+        application.application_data.delete('mentor_fav_project1') if fav1 == 'false'
+        application.application_data.delete('mentor_fav_project2') if fav2 == 'false'
```

Comment:
Yep, will for sure do a check :v:
Will do tonight though, when I'm back on my home machine.